18F/identity-idp
# LG-11759: Add inline device nickname editing for security keys

## 🎫 Ticket

[LG-11759](https://cm-jira.usa.gov/browse/LG-11759)

## 🛠 Summary of changes

Adds a rename/deletion interaction for Security Keys, similar to and extending what was introduced with [LG-11454](https://cm-jira.usa.gov/browse/LG-11454) (#9764).

## 📜 Testing Plan

1. Go to http://localhost:3000
2. Sign in
3. (If you don't already have a Security Key associated with your account, add one now)
4. On account dashboard, click "Manage" for your Security Key authenticator
5. Observe that you are able to manage the authenticator inline
6. Disable JavaScript and repeat steps 4 and 5, verifying that there is feature parity in the no-JavaScript experience

## 👀 Screenshots

![image](https://github.com/18F/identity-idp/assets/1779930/2e4d1efb-740d-4cd3-af8e-112aa3d97871)
